envs/main.yml

```diff
@@ -28,5 +28,4 @@
     - pyyaml=3.12
     - snakemake=5.19.0
     - unzip=6.0
-
```

Comment:
If the other YAML files have space here then please leave it as it was, for consistency.